core/domain/user_services.py

```diff
@@ -406,7 +406,10 @@ def record_user_started_state_editor_tutorial(user_id):
     _save_user_settings(user_settings)
 
 
-def update_email_preferences(user_id, can_receive_email_updates):
+def update_email_preferences(
```

Comment:
Just to clarify -- say a user has set their membership preference to False. Then they change "can_receive_email_updates" through their Preferences page. Wouldn't this reset their membership preferences to True, given how this function is implemented?

Instead, perhaps there ought to be two separate functions, one for each type of update.